packages/plugin-metrics/src/metrics.js

```diff
@@ -26,16 +26,75 @@ const Metrics = SparkPlugin.extend({
     return this.batcher.request(Object.assign({key}, value));
   },
 
-  submitClientMetrics(eventName, props) {
+  /**
+   * This corresponds to #sendSemiStructured() in the deprecated metrics handler
```

Comment:
@saurjai3 just giving you a heads up. once carter's PR goes in, the method in the new sdk will be called `submitClientMetrics` instead of `sendSemiStructured()`